applications/serial_lte_modem/overlay-carrier.conf

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_LWM2M_CARRIER=y
+
+# Workaround for known issue NCSDK-16916.
+# Set the size of the storage partition to a multiple of CONFIG_NRF_SPU_FLASH_REGION_SIZE.
+CONFIG_LWM2M_CARRIER_STORAGE_AREA_SIZE=0x8000
+
+# LwM2M carrier is only compiled for hard-float
+CONFIG_FP_HARDABI=y
+
+# The carrier library initializes the modem library
+CONFIG_NRF_MODEM_LIB_SYS_INIT=n
```

Comment:
This setting is no longer a requirement from the LwM2M carrier library, we could remove it from the overlay?